crates/revm/src/instructions/opcode.rs

```diff
@@ -164,7 +165,7 @@ impl OpCode {
     }
 
     #[inline(always)]
-    pub const fn u8(self) -> u8 {
+    pub const fn u8(&self) -> u8 {
```

Comment:
```suggestion
    pub fn u8(&self) -> u8 {
```
let it build but const probably needs to be removed